src/Integrations/class-amp.php

```diff
@@ -161,11 +159,11 @@ public static function construct_amp_json(): string {
 	public static function construct_amp_config(): array {
 		$options = self::$parsely->get_options();
 
-		if ( isset( $options['apikey'] ) && is_string( $options['apikey'] ) && '' !== $options['apikey'] ) {
+		if ( self::$parsely->site_id_is_set() ) {
 			return array(
 				'vars' => array(
 					// This field will be rendered in a JS context.
-					'apikey' => esc_js( $options['apikey'] ),
+					'site_id' => esc_js( $options['site_id'] ),
```

Comment:
Is it safe to replace this. I think it is not but I also don't have enough knowledge of AMP.

From here it seems like we are setting the key in JS context and it will used somehow by the users so we shouldn't changed it but please do confirm. Thanks

cc: @GaryJones

Reply:
Got the info from docs that it is necessary (https://docs.parse.ly/google-amp/)